corporate/tests/test_stripe.py

```diff
@@ -236,15 +236,17 @@ def normalize_fixture_data(
     ]
 
     # We'll replace "invoice_prefix": "A35BC4Q" with something like "invoice_prefix": "NORMA01"
+    # For patterns which can match multiple values like `[0-9]+`, include matching field in the translation
+    # to avoid false positives. See `exp_month` for example.
```

Comment:
This seems like a misunderstanding. The problem isn’t that the pattern matches multiple values, but rather that one of the values it matches is too generic, and then gets overzealously replaced elsewhere in the file (by `file_content = file_content.replace(match, normalized_values[pattern][match])`).

For example, after `"exp_month": ([0-9]+)` matches `"exp_month": 1`, we then replace `1` in the entire file, but we do not replace `2`.

Reply:
Yeah, I understood the problem but should have explained it like you did here. Will change the comment. Thanks!